test/extended/util/OWNERS

```diff
@@ -0,0 +1,4 @@
+reviewers:
+  - dmage
+approvers:
+  - dmage
```

Comment:
fyi i ended up moving this file down into `test/extended/util/annotate/generated` to make it more fine grained/specific, in master.

but given that this is 4.7 and won't be around that much longer, i'm not going to worry about it here.

Reply:
yeah, but it's hard to explain to cherrypick-bot :)